ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1327,15 +1327,8 @@ void GCS_MAVLINK::handleMessage(mavlink_message_t* msg)
                     result = MAV_RESULT_FAILED;
                     break;
                 }
-                // this blocks
-                float trim_roll, trim_pitch;
-                AP_InertialSensor_UserInteract_MAVLink interact(this);
-                if(copter.ins.calibrate_accel(&interact, trim_roll, trim_pitch)) {
-                    // reset ahrs's trim to suggested values from calibration routine
-                    copter.ahrs.set_trim(Vector3f(trim_roll, trim_pitch, 0));
-                    result = MAV_RESULT_ACCEPTED;
-                } else {
-                    result = MAV_RESULT_FAILED;
+                if(!copter.motors.armed()) {
```

Comment:
Isn't this if not needed? The first thing done when receiving this message is checking if motors are armed.

Also doesn't the result need to be set? (this question also applies to Rover and Plane)